rites/roman1969/src/models/calendar.ts

```diff
@@ -100,6 +105,7 @@ export class Calendar implements BaseCalendar {
       endOfSeason: endOfSeason ? endOfSeason.toISOString().substr(0, 10) : '',
       startOfLiturgicalYear: startOfSeasonsDic[Season.Advent].toISOString().substr(0, 10),
       endOfLiturgicalYear: endOfSeasonsDic[Season.OrdinaryTime].toISOString().substr(0, 10),
+      ...(isChristmasTime ? { isNativityOfTheLordOnWeekday: nativityOfTheLordDow !== 0 } : {}),
```

Comment:
seems like a decent solution...

Reply:
Decent as in you could improve it? :thinking: